modules/virtual_environments/auto-venv/venv_helpers.nu

```diff
@@ -23,6 +23,20 @@ export def venv-is-active [] {
     '__auto_venv' in (overlay list)
 }
 
+export def 'venv-create' [] {
+# Creates a virtual environment under the current directory
+	let venv_path = (pwd)
+	let venv_name = ((pwd) | path split | last) 
+	python3 -m venv .venv --clear --prompt $venv_name
```

Comment:
`python3` as binary name is quite specific to Linux distribution packages.
I wonder if that could be done a bit more general

Reply:
yeah, maybe @hyiltiz if you can find a way to make that more general? :thinking: